hipcheck/src/analysis/result.rs

```diff
@@ -0,0 +1,136 @@
+use crate::hc_error;
+use crate::report::Concern;
+use crate::Result;
+use crate::F64;
+
+/// Represents the enhanced result of a hipcheck analysis. Contains the actual outcome
+/// of the analysis, plus additional meta-information the analysis wants to provide to
+/// HipCheck core, such as raised concerns.
+#[allow(dead_code)]
```

Comment:
I'd just make a module-level `#[allow(unused)]`, and then remove it when you're done, rather than item-specific `allow` annotations.